build.gradle.kts

```diff
@@ -161,9 +165,7 @@ publishing {
 }
 
 signing {
-	setRequired({
-		project.version != "unspecified" && gradle.taskGraph.hasTask("publishToSonatype")
-	})
+	isRequired = releaseBuild
```

Comment:
I think `hasTask` can stay out of `releaseBuild`, but I'm not sure about the required precondition here (see #483)?

Reply:
You're right, I changed that back.